backstage/README.md

```diff
@@ -0,0 +1,19 @@
+# Backstage
+
+The Backstage plugin adds a Open with Backstage button to the resource detail view. If
+the plugin detects that the page is inside a Backstage iframe, it will send a
+message to the parent page to open the resource in the Backstage UI.
+
+This is the default template README for [Headlamp Plugins](https://github.com/headlamp-k8s/headlamp).
```

Comment:
This README need mending below here (this text is from the default template).

Probably a quickstart on how to get started with it.